docs/environment.yml

```
@@ -9,7 +9,7 @@ dependencies:
   - llvmlite=0.40
   - numpy
   - numpydoc
-  - setuptools
+  - setuptools=64
```

```suggestion
  - setuptools<65
```
I'm not sure the availability of version 64 can be guaranteed, it's just anything that's less than 65 should be fine?

---

fixed